test/sai_test/sai_route_test.py

```diff
@@ -1795,3 +1795,266 @@ def runTest(self):
     
     def tearDown(self):
         super().tearDown()
+
+class SviRouteL3Test(T0TestBase):
+    """
+    Verify route to svi route(through next hop and neighbor).
+    """
+
+    def setUp(self):
+        """
+        Test the basic setup process.
+        """
+        super().setUp()
+
+    def runTest(self):
+        """
+        Make Sure route interface, neighbor, and route already exist in the config for VLAN20 SVI and its members
```

Comment:
"Make Sure route interface, neighbor, and route already exist in the config for VLAN20 SVI and its members"
this step is a precondition in test plan, but not test purpose
In test cases, in order to avoid misunderstanding, please add more descriptions.
**Also including previous added cases**